docs/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Quick description
+
+Please include a summary of the changes and the related issue or the improvement. 
+Please also include relevant motivation and context. List any dependencies that are required for this change if needed.
+
+...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Provide data and command line to test
+
+...
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project (mostly pep8 compliant)
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
This means we have to document running tests better. Especially how to scope which tests to run, where to check for logs, warnings, coverage, etc.

Reply:
We'll see later, this template is there as a reminder of everything that has to be done when you submit a PR.
Don't need to over think this right now, I think.